Source/Core/DolphinQt/FIFO/FIFOAnalyzer.cpp

```diff
@@ -484,11 +495,34 @@ void FIFOAnalyzer::UpdateDescription()
   }
   else if (*cmddata == OpcodeDecoder::GX_LOAD_CP_REG)
   {
+    u8 cmd = *(cmddata + 1);
```

Comment:
Locals should be marked as const where applicable. Ditto where applicable elsewhere.

Reply:
Most of these have been done, though I think some more in FIFOAnalyzer could be cleaned up even if I didn't directly change them.